querydsl_test.go

```diff
@@ -0,0 +1,138 @@
+package querydsl
```

Comment:
Recommendations for additional tests enthusiastically encouraged!

Reply:
The "args" for the clauses seem a little lacking. Is there support for required/optional args?
Though, I see those kinds of tests being one-to-one with the specific clauses, once they get added.
That is not a necessity ;)

Reply:
I haven't really figured out if there's a need for optional arguments yet, so I haven't added any handling for them. You're right it's the sort of thing that'll be driven by the needs of specific clauses though.